system-addon/test/unit/lib/Screenshots.test.js

```diff
@@ -49,6 +49,36 @@ describe("Screenshots", () => {
     });
   });
 
+  describe("#maybeCacheScreenshot", () => {
+    let link;
+    beforeEach(() => {
+      link = {__sharedCache: {updateLink: sinon.stub()}};
```

Comment:
You can get the link to update with `updateLink(prop, val) { link[prop] = val; }`